docs/python/formatting.md

```diff
@@ -0,0 +1,75 @@
+---
+Order: 4
+Area: python
+TOCTitle: Formatting
```

Comment:
@gregvanl  any reservations against creating a new page? happy to wait until after the release to get this change in if it'd be easier. I also changed the order of the other pages and left ContentId empty here, but LMK if I did something wrong 😊